src/__tests__/runScript.test.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+/* eslint-disable global-require */
+
+import runScript from '../runScript';
+
+jest.mock('../utils/shell');
+
+describe('ALL command', () => {
```

Comment:
Typo: RUN-SCRIPT command (please fix, and don't forget to update the corresponding snapshot)